src_test/ppx_deriving/test_intf.ml

```diff
@@ -1 +1 @@
-type a = [%import: Stuff.a]
+[%%import: type a = Stuff.a]
```

Comment:
That syntax `module%type a = Stuff.a` should work too but is currently not covered by any tests. What would be the best way to add tests for it? Should we duplicate every test or just convert one/some of them to that syntax?

Reply:
My opinion on that is the following: making sure that either `[%%import: type a = Stuff.a]` or `module type%import a = Stuff.a` does the right thing is `ppx_import`'s job. Making sure that the two syntaxes `[%%import: type a = Stuff.a]` and `module type%import a = Stuff.a` do the same thing is the compiler's job. So here, I'd say it's enough to test one of the two syntaxes and to assume that the other one does the same.

However, it might be nice to add one single test of the other syntax for two reasons: that way you notice if something changes in the compiler and people who use the tests to look up how the library works can see the alternative syntax (although, tbh, I'd just add it to the README).

Of course, that decision is up to the `ppx_import` maintainers though!